crates/libtiny_tui/src/config.rs

```diff
@@ -1,18 +1,25 @@
 // To see how color numbers map to actual colors in your terminal run
 // `cargo run --example colors`. Use tab to swap fg/bg colors.
 
+use libtiny_common::{ChanName, ChanNameRef};
 use serde::de::{self, Deserializer, MapAccess, Visitor};
 use serde::Deserialize;
 use std::fs::File;
 use std::io::Read;
 use std::path::Path;
+use std::str::FromStr;
 
-pub use termbox_simple::*;
+pub(crate) use termbox_simple::*;
```

Comment:
Huh, I wonder how this slipped through.. It makes no sense to export termbox stuff from the `config` module. I think we should be able to remove the `pub` entirely, but maybe not in this PR. In fact, reverting this might be good as it doesn't seem to be relevant to the feature.